### PR TITLE
Postgres db

### PR DESCRIPTION
**Pull request by ben-mulhern**
Implement Postgres database is the main thing.
I've also fixed the logging issue by adding sfl4j stuff to build.sbt
Also corrected the SQL for the table definition of user_detail.
Also commented out the currencyservice from the server because the rest of it is not present in the develop branch yet so it wouldn't compile.

### CODE REVIEW

**Comment by ben-mulhern** [comment]
We shouldn't merge this until everyone's happy they can access the db on my machine, and/or we can find a new home for the db.

File: src/main/sql/user_detail.sql [hunk 1]
```diff
@@ -14,9 +14,9 @@ CREATE TABLE user_detail
   
   --Constraints
   CONSTRAINT active_is_yes_or_no
-  CHECK active IN ('Y','N')
+  CHECK (active IN ('Y','N')),
 
   CONSTRAINT admin_is_yes_or_no
-  CHECK admin IN ('Y','N')
+  CHECK (admin IN ('Y','N'))
```
Comment (KarlFinn) [inline src/main/sql/user_detail.sql:20]: Is this just for formatting or does PostGres need slightly different syntax for SQL?

**Review by KarlFinn — COMMENTED** [review]
All seem like sensible changes to me

File: src/main/sql/user_detail.sql (hunk 1 above)
Comment (ben-mulhern) [reply src/main/sql/user_detail.sql:20]: It didn't like it without the brackets. Nor does DB2 to be fair. Maybe H2 was more relaxed.

**Review by williamthurley — APPROVED** [review]

**Pull request merged**